provider/datagen/src/transform/icuexport/collator/test.rs

```diff
@@ -179,3 +187,52 @@ fn test_sv() {
         check_expectations(&collator, &cases);
     }
 }
+
+#[test]
+fn test_nb_nn_no() {
+    let any_dyn_provider = get_provider();
+    let any_provider = any_dyn_provider.as_any_provider();
+    let provider = any_provider.as_downcasting();
+
+    let input = vec!["ü", "y", "å", "ø"];
+    let expected = &["y", "ü", "ø", "å"];
+
+    // Test "no" macro language without fallback (should equal expected)
+    let collator = Collator::try_new(&provider, locale!("no"), CollatorOptions::new()).unwrap();
+    let mut strs = input.clone();
+    strs.sort_by(|a, b| collator.compare(a, b));
+    assert_eq!(strs, expected);
+
+    // Test "und" without fallback (should NOT equal expected)
+    let collator = Collator::try_new(&provider, locale!("und"), CollatorOptions::new()).unwrap();
+    let mut strs = input.clone();
+    strs.sort_by(|a, b| collator.compare(a, b));
+    assert_ne!(strs, expected);
+
+    // Test "nb" without fallback (should fail to load)
+    match Collator::try_new(&provider, locale!("nb"), CollatorOptions::new()) {
+        Ok(_) => panic!("Should fail to create 'nb' without fallback enabled"),
+        Err(_) => (),
+    };
```

Comment:
Clippy wants to make this an `if let`, but you can use `expect_err`:

```suggestion
    Collator::try_new(&provider, locale!("nb"), CollatorOptions::new())
      .expect_err("Should fail to create 'nb' without fallback enabled");
```

Reply:
`expect_err` didn't work because `Collator` does not implement `Debug` (and I'm not interested in scope-creeping this PR to add such an impl)

Reply:
Understandable, although I believe we've said that all public types should be `Debug`, another thing we need to track before 1.0...

Reply:
Is there a way to track this with Clippy or some other way?

Reply:
There's a `missing_debug_implementations` rustc lint